app/models/registration/policy.rb

```diff
@@ -0,0 +1,20 @@
+module Registration
+  class Policy < ApplicationRecord
+    belongs_to :registration_campaign,
+               class_name: "Registration::Campaign",
+               inverse_of: :registration_policies
+
+    enum :kind, { institutional_email: 0,
+                  prerequisite_campaign: 1,
```

Comment:
Not sure why we need `prerequisite_campaign` on the model-level. Isn't it the case that _every_ talk registration requires a prior seminar registration?  So based on the type of the `Registrable` one can already infer if another campaign is needed, right?

Reply:
Yes, I will add that (in an upcoming PR that contains all the changes in the architecture book that arise as a consequencxe of this PR and its review).